test/helper/webauth/index.ts

```diff
@@ -66,7 +66,7 @@ export default async function authorize(
         request.continue();
       }
     });
-    await page.goto(url, { waitUntil: 'networkidle2' });
+    await page.goto(url);
```

Comment:
ran oauth tests locally with `headless: false` set here to launch chromium:
https://github.com/jsforce/jsforce/blob/796d828fb4f6905f332bf0752fd66143f220e570/test/helper/webauth/index.ts#L50

with `waitUntil: 'networkidle2'` it wasn't waiting properly for the login page to load. The default value (`load`) works fine.
https://pptr.dev/api/puppeteer.page.goto#remarks